src/gui/render_scalers.cpp

```diff
@@ -219,16 +219,16 @@ ScalerSimpleBlock_t ScaleNormalDh = {
 	GFX_CAN_8|GFX_CAN_15|GFX_CAN_16|GFX_CAN_32,
 	1,2,{
 {	NormalDh_8_8_L,		NormalDh_8_15_L ,	NormalDh_8_16_L ,	NormalDh_8_32_L },
-{	             0,		NormalDh_15_15_L,	NormalDh_15_16_L,	NormalDh_15_32_L},
-{	             0,		NormalDh_16_15_L,	NormalDh_16_16_L,	NormalDh_16_32_L},
-{	             0,		NormalDh_24_15_L,	NormalDh_24_16_L,	NormalDh_24_32_L},
-{	             0,		NormalDh_32_15_L,	NormalDh_32_16_L,	NormalDh_32_32_L},
+{	             nullptr,		NormalDh_15_15_L,	NormalDh_15_16_L,	NormalDh_15_32_L},
+{	             nullptr,		NormalDh_16_15_L,	NormalDh_16_16_L,	NormalDh_16_32_L},
+{	             nullptr,		NormalDh_24_15_L,	NormalDh_24_16_L,	NormalDh_24_32_L},
+{	             nullptr,		NormalDh_32_15_L,	NormalDh_32_16_L,	NormalDh_32_32_L},
 {	NormalDh_8_8_L,		NormalDh_9_15_L ,	NormalDh_9_16_L ,	NormalDh_9_32_L }
 },{
 {	NormalDh_8_8_R,		NormalDh_8_15_R ,	NormalDh_8_16_R ,	NormalDh_8_32_R },
-{	             0,		NormalDh_15_15_R,	NormalDh_15_16_R,	NormalDh_15_32_R},
-{	             0,		NormalDh_16_15_R,	NormalDh_16_16_R,	NormalDh_16_32_R},
-{	             0,		NormalDh_24_15_R,	NormalDh_24_16_R,	NormalDh_24_32_R},
-{	             0,		NormalDh_32_15_R,	NormalDh_32_16_R,	NormalDh_32_32_R},
+{	             nullptr,		NormalDh_15_15_R,	NormalDh_15_16_R,	NormalDh_15_32_R},
+{	             nullptr,		NormalDh_16_15_R,	NormalDh_16_16_R,	NormalDh_16_32_R},
+{	             nullptr,		NormalDh_24_15_R,	NormalDh_24_16_R,	NormalDh_24_32_R},
+{	             nullptr,		NormalDh_32_15_R,	NormalDh_32_16_R,	NormalDh_32_32_R},
```

Comment:
Indentation seems to be off with the rest?

Reply:
This is how the indentation looks in current `main`:

<img width="705" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/f284f79e-9d27-429d-bf5d-f1a072d43cc9">

After the reformat: 

<img width="833" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/223ad221-010a-44b5-b297-1fa9e5bf464c">

Can you please fix this up so the nice tabular layout is kept? Then enclose the tabular block with `// clang-format off` and `// clang-format on` to exclude it from the Clang formatter? Check out `int10_modes.cpp` for a concrete example.